app.mjs

```diff
@@ -0,0 +1,19 @@
+import express from 'express';
+import path from 'path';
+import { getCommitersByName } from './git-tool-providers/azure-dev-ops.mjs';
+
+const app = express();
+const port = 3000;
+
+app.get('/api/getCommitersByName', async (req, res) => {
+    let commiters = await getCommitersByName();
```

Comment:
could get bogged down pulling all commits or even the last chunk ~15k.

Reply:
I'm gonna put a limit in the UI and the ability to use a date range.

Reply:
Ok to merge?